lib/gui/modules/analytics.js

```diff
@@ -164,23 +87,39 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * @public
    *
    * @description
-   * This function logs an exception in TrackJS.
+   * This function logs an exception to error reporting services.
    *
    * @param {Error} exception - exception
    *
    * @example
    * AnalyticsService.logException(new Error('Something happened'));
    */
   this.logException = (exception) => {
-    if (_.every([
-      SettingsModel.get('errorReporting'),
-      isRunningInAsar(),
-      errors.shouldReport(exception)
-    ])) {
-      $window.trackJs.track(exception);
-    }
+    resinCorvus.logException(exception);
+  };
 
-    $log.error(exception);
+  /**
+   * @summary Enable analytics
+   * @function
+   * @public
+   *
+   * @example
+   * AnalyticsService.enable()
+   */
+  this.enable = () => {
+    resinCorvus.enable();
+  };
+
+  /**
+   * @summary Disable analytics
+   * @function
+   * @public
+   *
+   * @example
+   * AnalyticsService.disable()
+   */
+  this.disable = () => {
```

Comment:
It isn't a no-op. It will the exception to the console, but not send it to external services.